_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,122 @@
+- Manual testing and compliance checks
+
+<div style="display: block; width: inherit; height: 400px; text-align: center;">
+    <a href="/ITStrategy/assets/images/Battle_of_Borodino_1812.png" style="display: block">
+        <img src = "/ITStrategy/assets/images/Battle_of_Borodino_1812.png"
+        alt = "A painting of the Battle of Borodino during the invasion of Russia by France in the Napoleonic Wars"
+        style="width: 500px; height: inherit;"
+        />
+    </a>
+    <small > The Battle of Borodino </small>
+</div>
+<br/>
+
+Not only have we decreased our productivity by a factor of 6-10 but we have also created a large lead time which reduces our ability to respond to change. Mark Schwartz, a serial CIO including that of US Homeland Security who is now an Enterprise Strategist at Amazon Web Services, describes a perfect analogy of why this is detrimental in his book War and Peace and IT. If the book sounds familiar, that's because it's meant as a follow up to Leo Tolstoy's famous book, War and Peace. The main theme of War and Peace is Napoleon's invasion of Russia during the Napoleonic Wars. Napoleon would push deeper into Russia and the Russians would withdraw until finally the scene is set for a large battle called the Battle of Borodino. Napoleon would survey the land and instruct his direct reports to station themself and their units. Napoleon then withdraws about a mile away to a fort he had taken prior called Shevardino where he would command his troops. To command his troops Napoleon relies on messengers to describe the situation and then issue commands to his army. However, since Napoleon is a mile away from the battlefield, by the time the messengers deliver the news it is already outdated and thus the commands that Napoleon issues do not reflect the current needs of his army. A famous example of this is when the bridge at Borodino is taken by Napoleon's army. A messenger is sent and delivers this news to Napoleon. So, Napoleon instructs that the army should cross the bridge and form ranks on the other side. However, what Napoleon didn't realize is by the time the messenger delivered the message, the bridge had not only been retaken but burned down such that it was no longer able to be crossed. This is very much the case with our current process, by the time a change is actually delivered, requirements have already changed and we are constantly playing a game of catch up. Manual processes, tests, checks and hand-offs also mean a larger potential of error and a greater risk of non-compliance for items such as accessibility and IT Security. There is simply too much to be able to check manually leading to overburdening our operational staff and a large amount of waiting and toil for developers to be able to incorporate change. This means we are focused on being reactive and putting out fires rather than looking at areas of improvement and addressing important issues such as technical debt.
```

Comment:
"There is simply too much to be able to check manually leading to overburdening our operational staff and a large amount of waiting..."

Perhaps a good place one could reference flow effeciency as a metric?

Ex: https://leankanban.com/flow-efficiency-a-great-metric-you-probably-arent-using/